app/apptesting/test_suite.go

```diff
@@ -396,6 +400,35 @@ func CreateRandomAccounts(numAccts int) []sdk.AccAddress {
 	return testAddrs
 }
 
+func TestMessageConcentratedLiquiditySerialization(t *testing.T, msg sdk.Msg) {
```

Comment:
Why are we defining concentrating liqudity-specific helpers in the global apptesting package?

Let's keep this appropriately scoped in the CL module please.

To build up on my other comment, I'm not sure why we need this custom helper and why we cannot reuse the one I linked